### PR TITLE
refactor(styling):code formatted using prettier

### PR DESCRIPTION
**Pull request by anjutshaju**

### CODE REVIEW

File: src/components/form-input/form-input.component.jsx [hunk 1]
```diff
@@ -2,16 +2,14 @@ import React from 'react';
 import './form-input.styles.scss';
 
 const FormInput = ({ handleChange, label, ...otherProps }) => (
```
Comment (amouxkin) [inline src/components/form-input/form-input.component.jsx]: Prefer `props` to `otherProps`.

File: src/components/sign-in/sign-in.component.jsx [hunk 2]
```diff
@@ -1,62 +1,64 @@
...
+            type="password"
+            value={this.state.password}
+            handleChange={this.handleChange}
+            required
+            label="Password"
+          />
+          <div className="buttons">
+            <CustomButton type="submit"> SignIn </CustomButton>
+            <CustomButton onClick={signInWithGoogle} isGoogleSignIn>
+              {' '}
```
Comment (amouxkin) [inline src/components/sign-in/sign-in.component.jsx]: Consider removing `{' '}`, these usually happen when using reformatting.
There are many instances of it, try to get them all.

File: src/components/form-input/form-input.component.jsx (hunk 1 above)
Comment (anjutshaju) [reply src/components/form-input/form-input.component.jsx]: Ok.

File: src/components/sign-in/sign-in.component.jsx (hunk 2 above)
Comment (anjutshaju) [reply src/components/sign-in/sign-in.component.jsx]: Ok.

**Review by amouxkin — APPROVED** [review]

**Comment by amouxkin** [comment]
@anjutshaju , please use semantic commits while committing as well.

**Review by Arjunmdas3211 — APPROVED** [review]

**Pull request merged**